tests/test_Parking.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from tornado import httpclient
+
+from simulation.simulation import ParkingLot
+import json
+
+# TODO add async http client
+
+@pytest.fixture
+def client():
+    return httpclient.AsyncHTTPClient()
+
+# def test_AddNew():
+#     lot = ParkingLot(1.0, 0.5, 500, "my_parking_lot", 1.5, 500, client)
+#     assert lot.data_to_send == json.dumps({"data": {"capacity": 500, "location": {"latitude": 1.0,
+#                                           "longitude": 0.5},
+#                                            "name": "my_parking_lot", "price": 1.5}
+#                                            })
+
+
+def test_AddZeroSpaces():
```

Comment:
If you re-use `parking.shared.rest_models.ParkingLot` then these tests would be unnecessary. :)